src/chunk_append/exec.c

```diff
@@ -15,6 +15,7 @@
 #include <parser/parsetree.h>
 #include <rewrite/rewriteManip.h>
 #include <utils/typcache.h>
+#include <utils/memutils.h>
```

Comment:
i think executor/executor.h should be included here instead which i'm already doing in another PR

Reply:
That's what I figured too, but it doesn't work because it appears the issue is that on some PG versions executor doesn't include memutils.h. I can include both, if that make sense, but only memutils.h won't quench the warning.

Reply:
since i'm adding executor.h in another pr this is fine, which compiler flags do you use since this wasn't picked up by our ci

Reply:
It happens on MacOS X with Clang so I guess that's why it is not picked up on CI.